--- Makefile.orig	2018-09-24 22:44:16.000000000 +0900
+++ Makefile	2018-09-24 23:48:55.000000000 +0900
@@ -13,7 +13,7 @@
 BDWGC_LIB = gc-7.6.4/.libs/libgc.a
 
 # GCC コンパイラオプション
-OPT_WARN = -Wall -Werror -Wno-unused-but-set-variable -Wno-maybe-uninitialized -Wno-format-security
+OPT_WARN = -Wall
 OPT_COMP = -std=gnu9x -g -I. -I../.. -I${BDWGC_DIR}/include -I../../${BDWGC_DIR}/include
 OPT_LINK = -lm -lreadline
 
